linux/Platform.h

```diff
@@ -27,6 +27,19 @@ in the source distribution for its full text.
    #define PATH_MAX 4096
 #endif
 
+#ifdef HAVE_LIBCAP
+   #define PLATFORM_LONG_OPTIONS \
+      {"drop-capabilities", optional_argument, 0, 128},
+   #define PLATFORM_LONG_OPTIONS_USAGE \
+         "   --drop-capabilities[=none|basic|strict] Drop Linux capabilities when running as root\n" \
+         "                                none - do not drop any capabilities\n" \
+         "                                basic (default) - drop all capabilities not needed by htop\n" \
+         "                                strict - drop all capabilities except those needed for core functionality\n"
```

Comment:
Prone to format string issues.

Reply:
I guess.  Any better alternative you can suggest?

Reply:
One could split the `printf()` call in `htop.c:printHelpFlag()` into 
```c
printf("%s " VERSION "\n"
         COPYRIGHT "\n"
         "Released under the GNU GPLv2.\n\n"
         "-C --no-color                   Use a monochrome color scheme\n"
         ...
         "-V --version                    Print version info\n",
         name);

Platform_printLongOptionsUsage(name);

printf("\n"
         "Long options may be passed with a single dash.\n\n"
         "Press F1 inside %s for online help.\n"
         "See 'man %s' for more information.\n",
         name, name);
```

but I am also fine with the current implementation, relying on `-Wformat` to alert on format string issues.

Reply:
@cgzones nice - I like that, it's cleaner and more flexible.  Will implement shortly and then merge, I think that's the last thing here.